src/Behat/Behat/Output/Statistics/TotalStatistics.php

```diff
@@ -76,7 +76,7 @@ public function resetAllCounters()
         $this->scenarioCounters = $this->stepCounters = array(
             TestResult::PASSED    => 0,
             TestResult::FAILED    => 0,
-            StepResult::UNDEFINED => 0,
+            TestResult::UNDEFINED => 0,
             TestResult::PENDING   => 0,
             TestResult::SKIPPED   => 0
```

Comment:
It might be interesting to perhaps use Enums (not int this PR, of course)?

Reply:
I don't think it is worth the BC break, especially given that our stats are using those test results as array keys, which is not possible with enums (as they are objects, which are invalid array keys)